src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -0,0 +1,1695 @@
+/**
+ * @file YarpSensorBridgeImpl.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_YARP_SENSOR_BRIDGE_IMPL_H
+#define BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_YARP_SENSOR_BRIDGE_IMPL_H
+
+#include <BipedalLocomotion/RobotInterface/YarpSensorBridge.h>
+#include <BipedalLocomotion/GenericContainer/Vector.h>
+
+// YARP os
+#include <yarp/os/Time.h>
+
+// YARP sig
+#include <yarp/sig/Vector.h>
+#include <yarp/sig/Image.h>
+
+// YARP Sensor Interfaces
+#include <yarp/dev/IAnalogSensor.h>
+#include <yarp/dev/IGenericSensor.h>
+#include <yarp/dev/MultipleAnalogSensorsInterfaces.h>
+
+// YARP Camera Interfaces
+#include <yarp/dev/FrameGrabberInterfaces.h>
+#include <yarp/dev/IRGBDSensor.h>
+
+// YARP Control Board Interfaces
+#include <yarp/dev/IEncodersTimed.h>
+#include <yarp/dev/IAxisInfo.h>
+
+// std
+#include <cmath>
+#include <set>
+#include <algorithm>
+
+using namespace BipedalLocomotion::RobotInterface;
+using namespace BipedalLocomotion::GenericContainer;
```

Comment:
You should not use `using namespace` in the header file. Here the explanation: https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/106/commits/4882805dd460666ec63b6d8d437f1d956c216a21